### PR TITLE
Update Grid Remote Validation app

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: grid/remote-validation/Client/Pages/Index.razor [hunk 1]
```diff
@@ -1,144 +1,154 @@
...
         }
     }
 
-    public async Task CreateHandler(GridCommandEventArgs args)
+    private async Task UpdateHandler(GridCommandEventArgs args)
     {
-        WeatherForecast currItem = args.Item as WeatherForecast;
+        WeatherForecast currItem = (WeatherForecast)args.Item;
+
+        ShouldRenderPopover = true;
```
Comment (ntacheva) [inline grid/remote-validation/Client/Pages/Index.razor:100]:
```suggestion
        
```
This variable is not declared anywhere and it's used only here. Do we need it?

File: grid/remote-validation/Client/Pages/Index.razor [hunk 2]
```diff
@@ -1,144 +1,154 @@
...
         }
         catch (Exception ex)
         {
-            // keep the grid in insert mode
+            // Keep the Grid in edit mode.
             args.IsCancelled = true;
-            // show information to the user
-            ShowErrorNotification(ex.Message);
+
+            await ShowErrorNotification(ex.Message);
```
Comment (ntacheva) [inline grid/remote-validation/Client/Pages/Index.razor:119]:
```suggestion
            ShowErrorNotification(ex.Message);
```
Cannot await void.

**Comment by ntacheva** [comment]
Maybe we can remove this image https://github.com/telerik/blazor-ui/blob/09f3774ea7341cef8db8bf1de929ca05b7d9f882/grid/remote-validation/Client/wwwroot/icon-192.png. It looks like we don't use it but I don't know why it was added.

**Review by ntacheva — DISMISSED** [review]

File: grid/remote-validation/Client/Pages/Index.razor [hunk 3]
```diff
@@ -1,144 +1,154 @@
...
     }
 
-    public async Task CreateHandler(GridCommandEventArgs args)
+    private async Task UpdateHandler(GridCommandEventArgs args)
     {
-        WeatherForecast currItem = args.Item as WeatherForecast;
+        WeatherForecast currItem = (WeatherForecast)args.Item;
+
+        
+
```
Comment (dimodi) [inline grid/remote-validation/Client/Pages/Index.razor:101]:
```suggestion
```

**Review by ntacheva — APPROVED** [review]

**Pull request merged**